app/views/machines/show.html.erb

```diff
@@ -1 +1,6 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+
```

Comment:
You have not defined @ machine anywhere. This needs to be defined in your MachinesController in the show method.